### PR TITLE
[stable/mongodb] add labels/annotations

### PR DESCRIPTION
**Pull request by vbehar**
### What this PR does / why we need it:

Add support for custom labels and annotations on the deployment or statefulsets, so that people can enable third-party kubernetes features that use either labels or annotations on the deployment/statefulset itself, and not the pods.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/mongodb/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 7.3.1
+version: 7.3.2
```
Comment (juan131) [inline stable/mongodb/Chart.yaml]: Please bump minor version instead.

File: stable/mongodb/README.md [hunk 2]
```diff
@@ -96,6 +96,8 @@ The following table lists the configurable parameters of the MongoDB chart and t
...
 | `replicaSet.pdb.maxUnavailable.secondary`          | PDB (max unavailable) for the MongoDB Secondary nodes                                                                                                     | `nil`                                                    |
 | `replicaSet.pdb.maxUnavailable.arbiter`            | PDB (max unavailable) for the MongoDB Arbiter nodes                                                                                                       | `nil`                                                    |
+| `annotations`                                      | Annotations to be added to the deployment or statefulsets                                                                                                 | `{}`                                                     |
+| `labels`                                           | Additional labels for the deployment or statefulsets                                                                                                      | `{}`                                                     |
```
Comment (juan131) [inline stable/mongodb/README.md:100]: Should we name these parameters `additionalLabels` and `additionalAnnotations`, respectively?

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for your contribution @vbehar ! Could you please take a look to my comments?
Please also rebase your PR from master since there are conflicts to be resolved. Thanks in advance

File: stable/mongodb/README.md (hunk 2 above)
Comment (vbehar) [reply stable/mongodb/README.md:100]: We can, if you prefer. I find `labels` & `annotations` shorter and easier to use, and these names are already used on lots of existing charts.

Comment (juan131) [reply stable/mongodb/README.md:100]: It's a minor thing. I don't have a strong opinion against `labels` & `annotations`

**Review by juan131 — APPROVED** [review]

**Pull request merged**